python/tvm/contrib/cutlass/gemm_operation.py

```diff
@@ -369,7 +369,8 @@ def instantiate_gemm_template(attrs):
             {
                 "bias_decl": "void* ptr_bias = (void*)(${bias_arg}->data);\n",
                 "ptr_c": "ptr_bias",
-                "c_stride": "${bias_arg}->ndim == 1 ? 0 : " + attrs["ldc"],
+                # "c_stride": "${bias_arg}->ndim == 1 ? 0 : " + attrs["ldc"],
```

Comment:
```suggestion

```

Reply:
is it needed?

Reply:
I see. So matmul+add(residual) is implemented using bias now?

Reply:
That's kind of strange here. Why don't we use beta=0 and residual epilogue, or Am I understanding it incorrectly?

Reply:
yes, otherwise there is an ambiguity in which of bias add or residual add should be used for `matmul + add` pattern. This causes a subtle issue for `FusesOpsByPattern`, which applies rewriting over a list of patterns which should be ordered by the "size" of a pattern. 

We are using a residual add pattern only for matmul + bias_add + residual add (two add).